Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityEyeGazeProvider.cs

```diff
@@ -12,15 +12,25 @@ namespace Microsoft.MixedReality.Toolkit.Input
     public interface IMixedRealityEyeGazeProvider : IMixedRealityGazeProvider
     {
         /// <summary>
-        /// Whether eye gaze is valid. It may be invalid due to timeout or lack of tracking hardware or permissions.
+        /// Whether eye tracking data is currently been used for gaze rather then head pose.
         /// </summary>
-        bool IsEyeGazeValid { get; }
+        bool IsGazeInputEyeBased { get; }
+
+        /// <summary>
+        /// Whether eye tracking data is available. It may be unavailable due to timeout or lack of tracking hardware or permissions.
+        /// </summary>
+        bool IsEyeTrackingAvailable { get; }
 
```

Comment:
Confusing. Why did you rename "IsEyeGazeValid" to "IsEyeTrackingAvailable"? Eye tracking data may be available, but not valid (e.g., during blinks).